lib/tictactoe/cli/main.ex

```diff
@@ -1,14 +1,40 @@
 defmodule TicTacToe.CLI.Main do
   alias TicTacToe.Board
+  alias TicTacToe.ConsoleIO
+  alias TicTacToe.Game
+  alias TicTacToe.Player
 
-  import TicTacToe.CLI.Console
+  @default_game %Game{
+    board: %Board{cells: Enum.to_list(1..9)},
+    players: %{
+      player1: %Player{
+        marker: "X",
+        name: "Player 1",
+        turn_msg: "Player 1's turn:",
+        type: :human
+      },
+      player2: %Player{
+        marker: "O",
+        name: "Player 2",
+        turn_msg: "Player 2's turn:",
+        type: :human
+      }
+    },
+    current_player: %Player{
+      marker: "X",
+      name: "Player 1",
+      turn_msg: "Player 1's turn:",
+      type: :human
+    },
+    markers: ["X", "O"],
+    size: 9
```

Comment:
> Instead of defining markers and size, what do you think about grabbing that information from the Players and Board respectively?

Can a value in a Struct reference another value inside that Struct, or would I have to define the Player outside the Game struct to grab its marker?

Reply:
If you choose to keep the `markers` and `size` keys, you would have to define the `Board` and `Player`s outside the `Game` struct first.

Since we already have this information from the `Board` and `Player`s, another approach could be to remove `size` and `marker` from the `Game` struct and wherever those values are expected, we just pass the length of the `Board` and a list of each player's marker respectively.

Reply:
Ah that makes so much sense, thanks!